app/views/products/index.html.haml

```diff
@@ -3,7 +3,7 @@
     %li
       %a{href: "#"} Cart
     %li
-      %a{href: "#"} Login
+      %a{href: "users/sign_in"} Login
```

Comment:
Is this for logging in or signing up?

Reply:
will you be able to see this link when you are logged in?

Reply:
Our idea is to have only one button, the "Login", this will direct to the Login view, and in this view there's a sign up button, so if the user doesn't have an account yet he can sign up from there. 

And when a user is logged in, he sees a greeting and a sign out button, we added that in the /views/layouts/application.html.haml >
- if user_signed_in?
    %p
      Hello, #{current_user.name} - #{link_to "Sign_out", destroy_user_session_path, method: 'delete'}

Reply:
okay fair enough